config/qa.yaml

```diff
@@ -57,6 +57,10 @@ NYPL_LOCATIONS_BY_CODE: https://nypl-core-objects-mapping-qa.s3.amazonaws.com/by
 # API_CLIENT_ID and API_CLIENT_SECRET must be configured in secrets file
 NYPL_API_CLIENT_TOKEN_URL: https://isso.nypl.org/oauth/token
 
+# DRB API Credentials
+DRB_API_HOST: '0.0.0.0'
```

Comment:
Same comment as above but for QA

Reply:
I'll change it to the host name to `drb-qa.nypl.org`